riscv-sbi.adoc

```diff
@@ -1674,6 +1677,388 @@ The possible error codes returned in `sbiret.error` are shown in the
+These are used to store measurements created and passed by the workload itself,
+through a <<measurement-extend-function>> call. This is set to 0 if the platform
+does not allow for extending the TCB  measurement.
+* `MSMT_REGS_INFO`: An array of measurement register descriptors, as defined by
+<<table_measurement_register_format>>. The size of this array is dynamic, and
+depends on the number of static and runtime registers.
+
+[#table_measurement_register_descriptor]
+.Measurement Register (MSMT_REG) Descriptor
+[cols="5,2,1,2,5", width=80%, align="center", options="header"]
+|===
+| Name            | Offset (bytes) | Type     | Size (bytes) | Description
+| TCB_LAYER_INDEX | 0              | Integer  | 1            | `DiceTcbInfo` layer index
+| FWID_INDEX      | 1              | Integer  | 1            | `DiceTcbInfo` `FWIDLIST` index
+| TCG_PCR_INDEX   | 2              | Integer  | 1            | https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClient_PFP_r1p05_v23_pub.pdf[`TCG PCR`] index mapping
+| RUNTIME_MSMT    | 3              | Boolean  | 1           a| * `0`: Static measurement register
+                                                               * `1`: Runtime measurement register
+|===
+
+The `MSMT_REG` fields are:
+
+* `TCB_LAYER_INDEX`: The `DiceTcbInfo` layer index in a `DiceMultiTcbInfo` X.509
+extension formatted attestation evidence. This field is only relevant when using
+the `DiceMultiTcbInfo` format for reporting attestation evidences.
+* `FWID_INDEX`: The `DiceTcbInfo` `FWIDLIST` index for this measurement
+register. This field is only relevant when using either the `DiceMultiTcbInfo`
+or the `DiceMultiTcbInfo` format for reporting attestation evidences.
+* `TCG_PCR_INDEX`: The https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClient_PFP_r1p05_v23_pub.pdf[`TCG PCR`]
+index the measurement registers map to.
+* `RUNTIME_MSMT`: Defines if the register is part of the static or runtime
+measurement registers set. A runtime measurement register is extensible, while a
+static one is not.
+
+The possible error codes returned in `sbiret.error` are shown in the
+<<table_attestation_capabilities_errors>> below.
+
+[#table_attestation_capabilities_errors]
+.Attestation Capabilities Errors
+[cols="2,3", width=90%, align="center", options="header"]
+|===
+| Error code            | Description
+| SBI_SUCCESS           | The attestation and measurement capabilities were
+                          succesfully reported.
+| SBI_ERR_OVERFLOW      | The attestation and measurement capabilities is larger
+                          than `caps_size`
+|===
+
+
+[#get-evidence-function,reftext="Get Evidence"]
+=== Function: Get Evidence (FID #1)
+
+[source, C]
+----
+struct sbiret sbi_attestation_get_evidence(unsigned long cert_request_addr,
+                                           unsigned long cert_request_size
+                                           unsigned long request_data_addr,
```

Comment:
nitpick: /s/request_data_addr/data_request_addr - Just to have similar format for argument names

Reply:
This needs to be marked as resolved.